pkg/controlplane/instance_test.go

```diff
@@ -399,68 +399,6 @@ func TestStorageVersionHashes(t *testing.T) {
 	}
 }
 
-func TestStorageVersionHashEqualities(t *testing.T) {
```

Comment:
We're running out of API groups that have both beta and GA levels.

Reply:
the check against actual enumerated hashes for served versions seems sufficient to me